content/presence-occupancy/index.textile

```diff
@@ -0,0 +1,23 @@
+---
+title: Overview
+meta_description: "Client presence is available for channels. Presence events occur when clients enter, leave, or change state on a channel and occupancy provide high level statistics."
+---
+
+Ably's presence feature enables clients or devices to announce their presence on a channel and the occupancy feature returns metrics related to the currently attached clients. Other devices or services may then subscribe to presence or occupancy in real time using our "realtime interface":/api/realtime-sdk, get one off updates with our "REST interface":/api/rest-sdk or integrate with webhooks "Ably Integrations":/integrations.
+
+h2(#presence-vs-occupancy). Occupancy vs. presence
```

Comment:
I think we can merge this section and the following one (use cases) with the rewrite to the introduction. Let's sync on this next week.